### PR TITLE
Adding Test Cases for Calibration Classifier

### PR DESCRIPTION
**Pull request by adityajain93**
The following PR added test cases for classification_calibration.py 
The test cases cover the following use cases
1. The classifier runs in all fit_modes and method types
2. Adding that there is improvement in Brier score after calibration 
and the predicted class is unchanged

### CODE REVIEW

File: tests/test_ClassificationCalibration.py [hunk 1]
```diff
@@ -0,0 +1,81 @@
+import logging
+
+from sklearn.metrics import brier_score_loss
+from sklearn.utils._testing import (
+    assert_array_equal,
+)
+import tensorflow as tf
```
Comment (pronics2004) [inline tests/test_ClassificationCalibration.py:7]: I guess we don't need this?

File: tests/test_ClassificationCalibration.py [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+from tests.test_utils import create_train_test_prod_split, split
+import unittest
+from unittest import TestCase
+from uq360.algorithms.classification_calibration import ClassificationCalibration
+tf.get_logger().setLevel(logging.ERROR)
```
Comment (pronics2004) [inline tests/test_ClassificationCalibration.py:12]: you could delete this?

File: tests/test_ClassificationCalibration.py [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+        returns pre-trained model object
+        """
+        from sklearn.neural_network import MLPClassifier
+        model = MLPClassifier(random_state = 42)
+        model.fit(x, y)
+        return model
+
+
+if __name__ == '__main__':
+    unittest.main()
```
Comment (pronics2004) [inline tests/test_ClassificationCalibration.py:81]: could you add a newline?

**Review by pronics2004 — COMMENTED** [review]
Thanks @adityajain93 ! 

Looks good, left a few minor comments.

**Comment by pronics2004** [comment]
The build failure seems to be from the latest TensorFlow version not related to this PR. 

Could you update the `requirements.txt`  to restrict tf to :
`tensorflow<=2.6`

**Comment by adityajain93** [comment]
Hey @pronics2004, I changed the requirements but the build still fails. Maybe we should separate the ticket and the build failure issue?

**Comment by pronics2004** [comment]
> Hey @pronics2004, I changed the requirements but the build still fails. Maybe we should separate the ticket and the build failure issue?

@adityajain93 Could you merge the latest `main`. This PR #20 fixed the TF version issue for now.

**Review by pronics2004 — APPROVED** [review]

**Pull request merged**